packages/api/src/beacon/routes/lodestar.ts

```diff
@@ -75,6 +79,16 @@ export type LodestarNodePeer = NodePeer & {
 
 export type LodestarThreadType = "main" | "network" | "discv5";
 
+const HistoricalSummariesResponseType = new ContainerType(
+  {
+    historicalSummaries: ssz.capella.HistoricalSummaries,
```

Comment:
@acolytec3 I converted this to lower case to align it with existing json responses